php/commands/src/CLI_Command.php

```diff
@@ -488,6 +488,27 @@ public function cmd_dump() {
 		echo json_encode( $this->command_to_array( EE::get_root_command() ) );
 	}
 
+	/**
+	 * Uninstalls easyengine completely along with all sites
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--yes]
+	 * : Do not prompt for confirmation.
+	 *
+	 * @subcommand self-uninstall
+	 */
+	public function self_uninstall( $args, $assoc_args ) {
+
+		EE::confirm("Are you sure you want to remove EasyEngine and all its sites(along with their data)?\nThis is an irreversible action. No backup will be kept.", $assoc_args);
+
+		Utils\default_launch("docker rm -f $(docker ps -aqf label=org.label-schema.vendor=\"EasyEngine\")");
+		$home = Utils\get_home_dir();
+		Utils\default_launch("rm -rf $home/.ee/");
+		Utils\default_launch("rm -rf $home/ee-sites/");
```

Comment:
Please don't hardcode `ee-sites`. This path is configurable through the config file variable `sites_path` and can vary.

This is the main reason why we are storing `site_root` in the database, because if the path was `a` in the beginning, there may be some sites inside `a` directory. When it gets changed by the user to `b`, the sites made after it can be in `b` directory and so on.

The best approach is to query the database and then remove all the site-roots you obtain from them. Note that, if by doing so you leave the parent directory like `~/ee-sites` as empty and not removed, then it is okay to leave empty dir after uninstallation.